lxd/instance/drivers/driver_qemu.go

```diff
@@ -1207,6 +1207,16 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 		}
 	}()
 
+	// Define a set of files to open that must wait until QEMU is called to be closed.
+	openFiles := make([]*os.File, 0)
```

Comment:
Why do we need another list of os.Files, the existing fdFiles list is kept open until qemu is started and then closed afterwards?

https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu.go#L1200-L1208

Reply:
@tomponline you are beating me in doing the review :) But I was also confused about the difference to `fdFiles`.

Reply:
`fdFiles` includes all the files that qemu has to have file descriptors for, as can be seen [here](https://github.com/canonical/lxd/blob/main/lxd/instance/drivers/driver_qemu.go#L1658). `openFIles` are just the directories that are opened to get a shorter path to a socket. But indeed all the files that are included in `openFiles` are also included in `fdFIles` so it doesn't serve any purpose.